### PR TITLE
Add ColorCheckboxList component

### PR DESCRIPTION
**Pull request by itsadesigning**

### CODE REVIEW

File: src/components/ColorCheckbox.vue [hunk 1]
```diff
@@ -37,15 +43,15 @@ export default {
       };
     },
     activeState() {
-      return this.checked ? {
+      return this.value === this.active ? {
```
Comment (aldinezi) [inline src/components/ColorCheckbox.vue]: As far as I can notice, this broke the initial functionality of color checkbox. Now, standalone color checkbox can not be toggled.

Comment (itsadesigning) [reply src/components/ColorCheckbox.vue]: Thanks for the input, I'll find a better solution!

File: src/components/ColorCheckboxList.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        '#dd1f1f',
+        '#ffe400'
+      ],
+      active: null
+    }
+  },
+  methods: {
+    selectCheckbox(value) {
+      this.active = value;
+      this.$emit('activeColor', this.colors[value]);
```
Comment (dika-pro) [inline src/components/ColorCheckboxList.vue]: Could you please use standard event for emitting changes? See https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components.

E.g.

```
this.$emit('change', this.colors[value]);
```

This way we could attach `v-mode` to your component.
Additionally you must handle `v-mode` on `beforeMount()` and set default active element.

e.g.

```
 beforeMount() {
    this.active = this.value;
  },
```

Comment (itsadesigning) [reply src/components/ColorCheckboxList.vue]: Will do, thanks for the input!

File: src/components/ColorCheckboxList.vue [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+export default {
+  props: {
+    type: String,
+    value: {
+      type: Object,
+      default: () => ({
+        key: 0,
+      }),
+    },
+    length: Number,
```
Comment (aldinezi) [inline src/components/ColorCheckboxList.vue]: Can you please add `type` and `default` keys for all props?

**Pull request merged**